src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.html

```diff
@@ -418,10 +414,8 @@
                                   class="btn btn-light"
                                   [cdPasswordButton]="'password' + ii">
                           </button>
-                          <button type="button"
-                                  class="btn btn-light"
-                                  [cdCopy2ClipboardButton]="'password' + ii">
-                          </button>
+                          <cd-copy-2-clipboard-button source="'password' + ii">
```

Comment:
I think this `source="'mutual_password' + ii"` will only work when using 2-way binding. I guess it should be
```suggestion
                          <cd-copy-2-clipboard-button source="{{ 'password' + ii }}">
```
or
```suggestion
                          <cd-copy-2-clipboard-button [source]="'password' + ii">
```

Reply:
Good note!~ Change effected